docs/_future/migrate_existing_app.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Migrate an existing app
+order: 7
+slug: migrate-existing-app
+lang: en
+layout: tutorial
+permalink: /tutorial/migrate-existing-app
+---
+
+## Migrate an existing app <span class="label-beta">BETA</span>
```

Comment:
Curious,are you able to repro this unusual formatting?
![Screen Shot 2022-11-02 at 2 39 36 PM](https://user-images.githubusercontent.com/55667998/199607684-6e054c59-5ede-42cd-b456-24d9a37213c2.png)

Reply:
Yess, I have not written this guide yet, I will get on it today